src/stories/Filter/FilterPage.tsx

```diff
@@ -0,0 +1,105 @@
+import { FC } from 'react';
+import Filter from 'src/Filter/Filter';
+import styled from '@emotion/styled';
+import { ThemeProvider } from '@emotion/react';
+import theme from 'src/styles/theme';
+import { useFilter } from 'src/Filter/util';
+import { ContextSwitchMenu, FilterBar, FilterJSONConfirmationModal, FilterJSONInput } from './components';
+import Input from 'src/Input/Input';
+import { FilterComponentProps } from 'src/Filter/types';
+
+const StyledInput = styled(Input)({
+    width: '100%'
+});
+
+const BASE_FILTER = {
+    getApiQueryUrl: (key, value) => {
+        return value ? `&${key}=${encodeURIComponent(value)}` : '';
+    },
+    getApiPostBody: (key, value) => (value ? { [key]: value } : undefined),
+    getBrowserQueryUrlValue: (value) => value,
+    getDefaultFilterValue: () => '',
+    isDefaultFilterValue: (value) => value === '',
+    getFilterBarLabel: (value) => value,
+    parseInitialFilterValue: (browserQueryUrlValue) => browserQueryUrlValue || '',
+    getFilterValueFromApiPostBody: (key, apiPostBody) => apiPostBody[key] || '',
+    renderComponent: ({ name, value, update }) => (
+        <StyledInput key={name} placeholder={name} onChange={(e) => update(e.target.value)} value={value} />
+    )
+};
+
+const FILTERS = {
+    keywords: {
+        description: 'Words to include.',
+        label: 'Keywords',
+        ...BASE_FILTER,
+        getDefaultFilterValue: () => 'lakehouse',
+        parseInitialFilterValue: (browserQueryUrlValue) => browserQueryUrlValue || 'lakehouse',
+        getFilterBarLabel: (value) => `Keywords: ${value}`
+    },
+    phrases: {
+        description: 'Phrases to lookup.',
+        label: 'Phrases',
+        ...BASE_FILTER,
+        getFilterBarLabel: (value) => `Phrases: ${value}`
+    }
+};
+
+const LOCATION = {
+    pathname: 'path',
+    search: '',
+    state: {
+        search: ''
+    },
+    hash: '',
+    key: 'key'
+};
+
+const DefaultWrapper = styled.div(({ theme }) => ({
+    backgroundColor: theme.colors.white,
+    fontFamily: '"Source Sans Pro", sans-serif',
+    margin: 0,
+    display: 'grid',
+    flexDirection: 'column',
+    alignItems: 'center',
+    border: theme.borders.primary
+}));
+
+const PageBody = styled.div(({ theme }) => ({
```

Comment:
Can move the styles in this file too to their own.

Reply:
@zach-jones-slalom - I'll move the styles. I was trying to keep the files in this folder pretty light since they are just for presentation purposes, but I guess if we decide to use them later they'll be easier to maintain/update with the refactor.

Reply:
@zach-jones-slalom  All files refactored.